### PR TITLE
Helm: add 'values' examples

### PR DESCRIPTION
**Pull request by heyleke**
##### SUMMARY
as described in https://github.com/ansible/ansible/issues/31117

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

### CODE REVIEW

File: lib/ansible/modules/cloud/misc/helm.py [hunk 1]
```diff
@@ -102,6 +103,7 @@
     state: present
     name: my-memcached
     namespace: default
+    values: {{ lookup('file', '/path/to/file/values.yaml') | from_yaml }}"
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/misc/helm.py]:
```suggestion
    values: "{{ lookup('file', '/path/to/file/values.yaml') | from_yaml }}"
```

File: lib/ansible/modules/cloud/misc/helm.py [hunk 2]
```diff
@@ -90,7 +90,8 @@
     state: present
     name: my-example
     namespace: default
-
+    values:
+      foo: "bar"
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/misc/helm.py:94]:
```suggestion
      foo: "bar"

```

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by heyleke** [comment]
@ansibot bot_status

**Review by Akasurde — APPROVED** [review]
LGTM

**Pull request merged**